sys/aarch64/pmap.c

```diff
@@ -414,10 +416,12 @@ void pmap_page_remove(vm_page_t *pg) {
     pmap_t *pmap = pv->pmap;
     vaddr_t va = pv->va;
     TAILQ_REMOVE(&pg->pv_list, pv, page_link);
-    TAILQ_REMOVE(&pmap->pv_list, pv, pmap_link);
-    pte_t *ptep = pmap_lookup_pte(pmap, va);
-    assert(ptep != NULL);
-    pmap_write_pte(pmap, ptep, 0);
+    WITH_MTX_LOCK (&pmap->mtx) {
```

Comment:
`pmap_page_remove` is called **only** from `pm_free_from_seg` which already holds global `physmem_lock`.